README.md

```diff
@@ -105,8 +105,10 @@ you want these variants to be:
 #### Performance Build Variants
 For accurate performance measurements, read this section!
 
+Before heading into performance, read this [quick guide on performance with Fenix](https://wiki.mozilla.org/Performance/How_to_get_started_on_Fenix)
+
 If you want to analyze performance during **local development** (note: there is a non-trivial performance impact - see caveats):
```

Comment:
Can we make this whole section friendlier and less nuanced with technical details, like we did with the performance guide? Some specific thoughts:
- we can mention Sentry, etc. as an afterthought
- Having "Recommendation", "Rationale", "Caveats" is really not friendly language :)
- We don't need to mention experiment opt-in yet
- We don't need to tell them it's slower to build

etc. etc.